starship.toml

```diff
@@ -3,7 +3,7 @@ style = "bold #ffb86c"
 
 [character]
 error_symbol = "[λ](bold #ff5555)"
-success_symbol = "[λ](bold #f8f8f2)"
+success_symbol = "[λ](bold #50fa7b)"
 
```

Comment:
This is then going to change your shell to always have a green lambda symbol, I’m not sure about this. Do you have some screenshots of before and after your proposed changes please

Reply:
Sure. But Starship uses green by default and the Dracula theme for Fish as well. 
* https://starship.rs/config/#character
* https://github.com/dracula/fish/blob/master/conf.d/dracula.fish

Before:
<img width="832" alt="Screenshot 2022-04-23 at 21 13 50" src="https://user-images.githubusercontent.com/1111056/164943160-e71db2b7-1279-48f6-a9f4-e03f772b3519.png">
After:
<img width="832" alt="Screenshot 2022-04-23 at 21 16 12" src="https://user-images.githubusercontent.com/1111056/164943163-72c60b84-e81d-4ee8-b627-37433b9b9870.png">